src/test/java/apicurito/tests/steps/VerificationSteps/OperationVerifications.java

```diff
@@ -0,0 +1,248 @@
+        collector.assertThat(rb.size()).as("Request body is not created!").isEqualTo(1);
+    }
+
+    @Then("^check that request body description is \"([^\"]*)\"$")
+    public void checkThatRequestBodyDescriptionIs(String expectedDescription) {
+        String description = OperationUtils.getOperationRoot().$(OperationSteps.REQUEST_BODY_SECTION).$("inline-markdown-editor").getText();
+        collector.assertThat(description).as("Request body description should be %s but is %s", expectedDescription, description).isEqualTo(expectedDescription);
+    }
+
+    @Then("^check that request body type is \"([^\"]*)\"$")
+    public void checkThatRequestBodyTypeIs(String expectedType) {
+        String type = OperationUtils.getOperationRoot().$(OperationSteps.REQUEST_BODY_SECTION).$(CommonUtils.DropdownButtons.PROPERTY_TYPE.getButtonId()).getText();
+        collector.assertThat(type).as("Type is %s but should be %s", type, expectedType).isEqualTo(expectedType);
+    }
+
+    @Then("^check that request body type of is \"([^\"]*)\"$")
+    public void checkThatRequestBodyTypeOfIs(String expectedOf) {
+        String of = OperationUtils.getOperationRoot().$(OperationSteps.REQUEST_BODY_SECTION).$(CommonUtils.DropdownButtons.PROPERTY_TYPE_OF.getButtonId()).getText();
+        collector.assertThat(of).as("Type of is %s but should be %s", of, expectedOf).isEqualTo(expectedOf);
+    }
+
+    @Then("^check that request body type as is \"([^\"]*)\"$")
+    public void checkThatRequestBodyTypeAsIs(String expectedAs) {
+        String as = OperationUtils.getOperationRoot().$(OperationSteps.REQUEST_BODY_SECTION).$(CommonUtils.DropdownButtons.PROPERTY_TYPE_AS.getButtonId()).getText();
+        collector.assertThat(as).as("Type as is %s but should be %s", as, expectedAs).isEqualTo(expectedAs);
+    }
+
+    @Then("^check that exist request form data$")
+    public void checkThatExistRequestFormData(DataTable table) {
+        CommonUtils.openCollapsedSection(OperationSteps.REQUEST_BODY_SECTION);
+
+        for (List<String> dataRow : table.cells()) {
+            checkRow(dataRow, OperationSteps.REQUEST_BODY_SECTION);
+        }
+    }
+
+    @Then("^check that exist query parameters$")
+    public void checkThatExistQueryParameter(DataTable table) {
+        CommonUtils.openCollapsedSection(OperationSteps.QUERY_PARAM_SECTION);
+
+        for (List<String> dataRow : table.cells()) {
+            checkRow(dataRow, OperationSteps.QUERY_PARAM_SECTION);
+        }
+    }
+
+    @Then("^check that exist header parameters$")
+    public void checkThatExistHeaderParameters(DataTable table) {
+        CommonUtils.openCollapsedSection(OperationSteps.HEADER_PARAM_SECTION);
+
+        for (List<String> dataRow : table.cells()) {
+            checkRow(dataRow, OperationSteps.HEADER_PARAM_SECTION);
+        }
+    }
+
+    private void checkRow(List<String> dataRow, By section) {
+        String rowType;
```

Comment:
If you add those 2 parameters into method arguments, you can then get rid of this whole switch statement.

Reply:
updated